internal/runner/options.go

```diff
@@ -339,3 +362,44 @@ func (options *Options) configureResume() error {
 	}
 	return nil
 }
+
+func (options *Options) configureQueryOptions() {
+	queryMap := map[string]*bool{
+		"a":     &options.A,
+		"aaaa":  &options.AAAA,
+		"cname": &options.CNAME,
+		"ns":    &options.NS,
+		"txt":   &options.TXT,
+		"srv":   &options.SRV,
+		"ptr":   &options.PTR,
+		"mx":    &options.MX,
+		"soa":   &options.SOA,
+		"axfr":  &options.AXFR,
+		"caa":   &options.CAA,
+		"any":   &options.ANY,
+	}
+
+	for _, qt := range options.QueryType {
+		if qt == "all" {
+			for _, val := range queryMap {
+				*val = true
+			}
+		} else {
+			if val, ok := queryMap[qt]; ok {
+				*val = true
+			}
+		}
+	}
+
+	for _, et := range options.ExcludeType {
+		if et == "all" {
```

Comment:
I think `all` should not be a recognized value for exclude query types, as it makes the tool not usable

Reply:
> Using both -q a -eq a makes the tool anyway sending out an a query (standard behavior)
I haven't touched this behavior, should we?

Reply:
I think we should keep the current behavior as base, but if `eq` exclude all other possibilities before adding `a` we can inform the user (ref: https://github.com/nmap/nmap/blob/ff92f5bae941058fae4782167fc7cb49b29c9440/scan_lists.cc#L460)